includes/rest/class-interaction.php

```diff
@@ -43,6 +43,24 @@ public static function register_routes() {
 				),
 			)
 		);
+
+		\register_rest_route(
+			ACTIVITYPUB_REST_NAMESPACE,
+			'/reactions/(?P<id>\d+)',
+			array(
+				'methods'             => \WP_REST_Server::READABLE,
+				'callback'            => array( self::class, 'get_reactions' ),
+				'permission_callback' => '__return_true',
+				'args'                => array(
+					'id' => array(
+						'required'          => true,
+						'validate_callback' => function ( $val ) {
+							return is_numeric( $val );
+						},
+					),
+				),
```

Comment:
We could just copy [what Core does for the post id](https://github.com/WordPress/wordpress-develop/blob/af1682d4d327655781bc0b18db0f5f31f06d9b2b/src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php#L114-L121). Setting the type to integer triggers the built-in validation and sanitization calls

Reply:
Even better!